### ISSUE TITLE
If you regenerate a long answer immediately after generating it, the second answer may not be fully displayed

### ISSUE THREAD

**Issue by gaowanliang** [ading2210/poe-api#3]
For example
## Code
```python
message = "I hope you can generate an article about the Pacific Ocean, about 300 words"
for chunk in client.send_message("capybara", message):
    pass
print(chunk["text"])
message = "I hope you can generate an article about the Atlantic Ocean"
for chunk in client.send_message("capybara", message):
    pass
print(chunk["text"])
```
## Output
```text
Certainly, here's an article about the Pacific Ocean:

# The Pacific Ocean: A Vast and Diverse Body of Water

The Pacific Ocean is the largest body of water on Earth, covering an area of approximately 63.8 million square miles. It stretches from the Arctic Ocean in the north to the Southern Ocean in the south, and from the western shores of North and South America to the eastern shores of Asia and Australia.

The Pacific Ocean is home to an incredible variety of marine life, from tiny plankton to massive whales, and from colorful coral reefs to deep-sea trenches. Some of the most famous marine animals in the world inhabit the Pacific, including great white sharks, blue whales, and sea turtles.

In addition to its diverse marine life, the Pacific Ocean is also home to a number of important islands and archipelagos, including Hawaii, Fiji, and the Galapagos Islands. These islands are not only important for their ecological significance, but also for their cultural and historical importance to the people who live on them.

The Pacific Ocean is also an important source of natural resources, including fish, oil, and minerals. Many countries around the Pacific Rim rely on the ocean for their livelihoods, and the Pacific is a major shipping route for goods traveling between Asia and North America.

Despite its vast size and importance, the Pacific Ocean is facing a number of threats, including overfishing, pollution, and climate change. Scientists and policymakers are working to address these challenges and protect the health of this important body of water for future generations.

One of the most significant environmental challenges facing the Pacific Ocean is plastic pollution. The ocean's currents have created vast areas of floating plastic waste, known as the Great Pacific Garbage Patch. This pollution not only harms marine life, but also poses a threat to human health, as microplastics are now found in seafood and drinking water.

Another challenge facing the Pacific Ocean is the acidification of its waters. As carbon dioxide from human activities dissolves in ocean waters, it lowers the pH of the water, making it more acidic. This can have devastating effects on marine ecosystems and the species that depend on them.

Despite these challenges, there is hope for the Pacific Ocean. Efforts are underway to address plastic pollution and reduce carbon emissions to mitigate the effects of climate change. Conservation measures are also being implemented to protect marine life and their habitats.

In conclusion, the Pacific Ocean is a vast and diverse body of water that is home to an incredible array of marine life, important islands and archipelagos, and valuable natural resources. Despite the challenges it faces, the Pacific remains one of the most important and awe-inspiring features of our planet. It is up to each of us to do our part to protect this vital resource for future generations.
Certainly, here's an article about the Atlantic 
```

**Comment by ading2210** [comment]
This has been fixed in version 0.1.3.

```python
import poe
import logging

poe.logger.setLevel(logging.INFO)

client = poe.Client("TOKEN")

message = "I hope you can generate an article about the Pacific Ocean, about 300 words"
for chunk in client.send_message("capybara", message):
  print(chunk["text_new"], end="", flush=True)

print()
message = "I hope you can generate an article about the Atlantic Ocean"
for chunk in client.send_message("capybara", message):
  print(chunk["text_new"], end="", flush=True)
print()
```

```
INFO:root:Downloading next_data...
INFO:root:Downloading channel data...
INFO:root:Downloading https://poe.com/_next/data/KyiVpGk8gbbnwLUpZdqrb/sage.json
INFO:root:Downloading https://poe.com/_next/data/KyiVpGk8gbbnwLUpZdqrb/gpt-4.json
INFO:root:Downloading https://poe.com/_next/data/KyiVpGk8gbbnwLUpZdqrb/claude+.json
INFO:root:Downloading https://poe.com/_next/data/KyiVpGk8gbbnwLUpZdqrb/claude.json
INFO:root:Downloading https://poe.com/_next/data/KyiVpGk8gbbnwLUpZdqrb/chatgpt.json
INFO:root:Downloading https://poe.com/_next/data/KyiVpGk8gbbnwLUpZdqrb/dragonfly.json
INFO:root:Sending message to capybara: I hope you can generate an article about the Pacific Ocean, about 300 words
The Pacific Ocean is the largest and deepest ocean on Earth, covering approximately one-third of the planet's surface. It is bounded by the continents of North and South America to the east, Asia and Australia to the west, and the Southern Ocean to the south. The ocean's name comes from the Latin word "pacificus," which means peaceful, as it was named by the Portuguese explorer Ferdinand Magellan, who crossed it in 1520 and found it to be calm.

The Pacific Ocean is home to a vast array of marine life, including whales, dolphins, sharks, and sea turtles, as well as numerous species of fish and invertebrates. It also contains many important ecosystems, including coral reefs, seagrass meadows, and deep-sea hydrothermal vents.

One of the most famous features of the Pacific Ocean is the Great Barrier Reef, a massive coral reef system off the coast of Australia that is home to thousands of species of marine life. The Pacific also contains many other important ecosystems, including kelp forests, seagrass meadows, and deep-sea hydrothermal vents.

The ocean's vast size and depth make it a vital part of the Earth's climate system. The Pacific's currents and temperatures influence weather patterns around the globe, and it plays a key role in regulating the planet's temperature and carbon cycle. The ocean is also a major source of food and resources for human populations around the world, with fisheries in the Pacific providing billions of dollars in revenue each year.

Despite its importance, the Pacific Ocean is facing a range of challenges. Overfishing, pollution, and climate change are all affecting the health of the ocean's ecosystems. Rising sea temperatures and ocean acidification are causing harm to coral reefs and other marine life, while plastic pollution and other forms of waste are threatening the ocean and its inhabitants.

Governments, NGOs, and individuals are working to protect and conserve the Pacific Ocean. Efforts are underway to reduce pollution, manage fisheries sustainably, and promote the conservation of marine ecosystems. By working together, we can ensure that the Pacific Ocean remains a vital part of our planet's natural heritage for generations to come.
INFO:root:Sending message to capybara: I hope you can generate an article about the Atlantic Ocean
The Atlantic Ocean is the second largest ocean on Earth, covering approximately 20% of the planet's surface. It is bounded by the continents of North and South America to the west, and Europe and Africa to the east. The ocean's name comes from the Greek mythological figure Atlas, who was said to hold up the sky and the earth.

The Atlantic Ocean is home to a diverse range of marine life, including whales, dolphins, sharks, and sea turtles, as well as numerous species of fish and invertebrates. It also contains many important ecosystems, including coral reefs, seagrass meadows, and deep-sea hydrothermal vents.

One of the most famous features of the Atlantic Ocean is the Gulf Stream, a powerful current that flows from the Gulf of Mexico to the North Atlantic. The Gulf Stream plays a vital role in regulating the Earth's climate, as it helps to distribute heat around the planet and influences weather patterns in Europe and North America.

The Atlantic Ocean has a rich history, and has played a key role in world exploration, trade, and conflict. The ocean was the site of numerous naval battles during World War II, and was also a major route for the transatlantic slave trade. Today, the Atlantic remains an important source of trade and transportation, with thousands of ships crossing the ocean every day.

However, like all oceans, the Atlantic is facing a range of challenges. Overfishing, pollution, and climate change are all affecting the health of the ocean's ecosystems. Rising sea temperatures and ocean acidification are causing harm to coral reefs and other marine life, while plastic pollution and other forms of waste are threatening the health of the ocean and its inhabitants.

Despite these challenges, there are many efforts underway to protect and conserve the Atlantic Ocean. Governments, NGOs, and individuals are working to reduce pollution, manage fisheries sustainably, and promote the conservation of marine ecosystems. By working together, we can ensure that the Atlantic Ocean remains a vital part of our planet's natural heritage for generations to come.
```
